Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -366,6 +355,28 @@ class AccountStatusSetting: WithAccountSetting {
     }
 }
 
+// MARK: - Hidden Settings
+// For great debugging!
```

Comment:
For great debugging? Please tell me more :P

Reply:
Haha, I added the Mark, not anything else. I'll fill it out. Good point.